### PR TITLE
Adding some automated testing Nightcrawler and Backstopjs

### PR DESCRIPTION
**Pull request by mrossi113**
- [x] Added the Nightcrawler to the site repository 
- [x] Add to CircleCI config.yml for only Nightcrawler
- [x] Add Backstopjs to use only locally 

**Test Nightcrawler (Errors and Performance Testing)**
To test the Nightcrawler `docker-compose run drupal node_modules/.bin/nightcrawler crawl` it will run Nightcrawler locally or view it in CircleCI `https://circleci.com/gh/LastCallMedia/Last-Call-Media/3775` look for Nightcrawler

**Test Backstopjs (Visual Regression Testing)**
To test Backstopjs either create some pages and add to the page.json or just run the following commands:
Baseline aka reference = `docker-compose run backstop reference`
test against baseline = `docker-compose run backstop test`
To view the results = `open backstop/report/index.html`

Jira ticket: https://lastcall.atlassian.net/browse/LCM-747

### CODE REVIEW

**Comment by rbayliss** [comment]
We just merged a version of this into the lastcallmedia.com codebase.  Let's give that a week to percolate and make sure it works well, then this is ready to roll.

File: .circleci/config.yml [hunk 1]
```diff
@@ -95,6 +95,8 @@ jobs:
       - run: {name: 'Refresh Site', command: 'composer site:import -- -c /tmp/db-cache'}
       - save_cache: {name: 'Save DB Cache', key: 'db-{{ checksum "/tmp/db-cache-indicator" }}', paths: ['/tmp/db-cache']}
       - run: {name: 'Wait for Apache', command: 'dockerize -wait tcp://localhost:80 -timeout 5s'}
+      - run: {name: 'Prepare for Nightcrawler', command: 'sleep 60' }
```
Comment (rbayliss) [inline .circleci/config.yml]: I'd like to get rid of this sleep before we merge.

**Comment by rbayliss** [comment]
I'm good with this once we remove the `sleep`!

File: .circleci/config.yml (hunk 1 above)
Comment (mrossi113) [reply .circleci/config.yml]: The sleep 60 was removed from the CircleCI config file.

**Pull request merged**